src/07_workarounds/05_async_in_traits.md

```diff
@@ -1,10 +1,10 @@
 # `async` in Traits
 
-Currently, `async fn` cannot be used in traits. The reasons for this are
-somewhat complex, but there are plans to remove this restriction in the
-future.
+Currently, `async fn` cannot be used in traits on the stable release of Rust.
+As of 17th Nov 22, an MVP of async-fn-in-trait is available on the nightly
```

Comment:
```suggestion
As of 17 November 2022, an MVP of async-fn-in-trait is available on the nightly
```

I don't know if we have a standard date format for this book already, but I'd suggest at least using a four digit year.

Reply:
I guess another option would be to reference the specific nightly version where this feature became available, like `nightly-2022-11-17` or whatever the version ended up being.

Reply:
Agree to improve date to be "human readable" yep, good idea, will push commit updating it!

I don't personally track specific versions of nightly (either use it if I have to, or stick to stable..) so not sure of how helpful that would be. Had a quick look to find "what feature landed in what version of nightly" information and didn't find it easily either.